GNUmakefile

```diff
@@ -16,6 +16,10 @@ else
 GOTEST_PKGS=$(shell go list ./... | sed 's/github.com\/hashicorp\/nomad/./' | egrep -v "^($(GOTEST_PKGS_EXCLUDE))(/.*)?$$")
 endif
 
+# tag corresponding to latest release we maintain backward compatibility with
+# NOTE: using a pre-release, as buf fails to build the latest official release now
+PROTO_COMPARE_TAG ?= v1.0.0-beta3$(if $(findstring ent,$(GO_TAGS)),+ent,)
```

Comment:
I tried to set the version to the latest release (e.g. 0.12.X) but had some  failures (due to the proto imports). I may follow up in another time to use it.  Ideally, this is the latest "official" release, as we don't care as much about compatibility against a beta release.